app/import_config/load_form_json.py

```diff
@@ -255,6 +238,11 @@ def insert_form_config(form_config, form_id):
     start_page_path = form_config["startPage"]
     form_section_list = create_form_sections_db(form_config)
 
+    list_names_to_ids = {}
+    for lizt in form_config.get("lists", []):
+        new_list = insert_list(do_commit=False, list_config={"is_template": True, **lizt})
```

Comment:
in a Json there might have unwanted lists so don't we need to remove them from the template I think better to have list items which is already in use ? better to check first usage of the list in component then create ? 

also once template is deleted is the list also going to be deleted ?

Reply:
I think we mentioned this when we were pairing on these changes - we are trying to represent the Form Designer schema as much as possible in our own data schema - if Form Designer schema allows unattached lists, then so should we. The basic principle is minimising complexity our end

Regarding template deletion and list deletion, this is a good question - maybe we can create a separate ticket for ensuring cascade delete across the data schema as an outcome from your excellent work on the [FAB code end-to-end understanding](https://mhclgdigital.atlassian.net/browse/FS-4900) ticket?